docs/sops/serverless-controller-knative-up.md

```diff
@@ -0,0 +1,149 @@
+# OpenShift Serverless Knative Serving Control Plane Readiness
+
+## Severity: Critical
+
+## Impact
+
+Any users or services using Knative Serving will be unable to get their Knative Services reconciled.
+This means, any new Knative Services might not be created, and any existing Knative Services might not be updated. Also, it might also mean that the user workload is not being served.
+
+## Summary
+
+OpenShift Serverless does not create any alerts out-of-the box. However, users can create alerts based on the metrics exposed.
+
+The metric `knative_up{namespace="openshift-serverless", type="serving_status"}` should always have a value of `1`. 
```

Comment:
Actually for gauges it is better to measure it over some period of time eg. https://www.robustperception.io/alerting-on-gauges-in-prometheus-2-0/.

Reply:
Thanks for the great input @skonto !

`avg_over_time(up{job="jobname"}[5m]) < 0.9` we should use this kind of structure in https://issues.redhat.com/browse/SRVCOM-2704

Reply:
Also updated the text here to reflect that `knative_up` is a gauge and consumers should measure it over period of time. However, we don't provide alerts in this PR. That's part of SRVCOM-2704.